internal/generator/vector/outputs.go

```diff
@@ -25,7 +27,18 @@ func Outputs(clspec *logging.ClusterLoggingSpec, secrets map[string]*corev1.Secr
 	ofp := OutputFromPipelines(clfspec, op)
 
 	for _, o := range clfspec.Outputs {
-		secret := secrets[o.Name]
+		var secret *corev1.Secret
+		if s, ok := secrets[o.Name]; ok {
+			secret = s
+			log.V(9).Info("Using secret configured in output")
+		} else {
+			secret = secrets[constants.LogCollectorToken]
+			if secret != nil {
+				log.V(9).Info("Using secret configured in logcollector-token")
+			} else {
+				log.V(9).Info("No Secret found in logcollector-token")
```

Comment:
"No secret found in " +  constants.LogCollectorToken ?

Reply:
updated